pages/task/read/index.js

```diff
@@ -17,6 +19,17 @@ module.exports = settings => {
     next();
   });
 
+  app.use('/', (req, res, next) => {
+    if (req.task.data.data.profileId && req.task.data.data.establishmentId) {
+      return req.api(`/establishment/${req.task.data.data.establishmentId}/profile/${req.task.data.data.profileId}`)
+        .then(({ json: { data } }) => {
+          req.profile = cleanModel(data);
+        })
+        .then(() => next())
+        .catch(next);
+    }
+  });
```

Comment:
what happens if `req.task.data.data.profileId && req.task.data.data.establishmentId` is falsy?

Reply:
hint: nothing, it will spin _forever_